contracts/container/contract.go

```diff
@@ -491,6 +494,143 @@ func List(owner []byte) [][]byte {
 	return list
 }
 
+// AddNextEpochNodes accumulates passed nodes as container members for the next
+// epoch to be committed using [CommitContainerListUpdate]. Nodes must be
+// grouped by selector index from placement policy (SELECT clauses). Results of
+// the call operation can be received via [Nodes]. This method must be called
+// only when a container list is changed, otherwise nothing should be done.
+// Call must be signed by the Alphabet nodes.
+func AddNextEpochNodes(cID interop.Hash256, placementVector int, publicKeys []interop.PublicKey) {
```

Comment:
Should we check the `placementVector`? `add(X, 25, []...)`, for example?

Reply:
> Should we check the placementVector? 

what exact check you mean? size? check that we have 0-24 indexes?

Reply:
0-24. Maybe it's a bit stretched, but well, it's a contract, it's the last guard before this (broken) data enters the system.